README.md

```diff
@@ -153,11 +149,11 @@ $ kubectl create -f deployments/sriov-crd.yaml
 This section explains an example deployment of SR-IOV Network device plugin in Kubernetes if you choose DANM as your meta plugin.
 
 ##### Install DANM
-Refer to [DANM documentation](https://github.com/nokia/danm#getting-started) for detailed instructions.
+Refer to [DANM deployment documentation](https://github.com/nokia/danm#deployment) for detailed instructions.
 
 ##### Create SR-IOV type networks
 DANM supports the Device Plugin based SR-IOV provisioning with the dynamic level.
-This means that all DANM API features seamlessly work together with the SR-IOV setup described above, whether you use the [lightweight](https://github.com/nokia/danm#lightweight-network-management-experience), or the [production grade](https://github.com/nokia/danm#production-grade-network-management-experience) network management APIs.
+Refer to the [DAMN User Guide documentation](https://github.com/nokia/danm#user-guide) for detailed instructions.
```

Comment:
nit: you can reference the user-guide directly

https://github.com/nokia/danm/blob/master/user-guide.md